lib/MusicBrainz/Server/Report/NotVariousArtists.pm

```diff
@@ -0,0 +1,47 @@
+package MusicBrainz::Server::Report::NotVariousArtists;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::RecordingReport',
+     'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
```

Comment:
Is there actually a way to subscribe to recordings? https://wiki.musicbrainz.org/Subscription says:
> It is possible to subscribe to artists, labels and editors.